### PR TITLE
[15.0][MIG] web_widget_x2many_2d_matrix

### PR DESCRIPTION
**Pull request by ChrisOForgeFlow**
Standard Migration
@ForgeFlow

### CODE REVIEW

File: web_widget_x2many_2d_matrix/__manifest__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     "name": "2D matrix for x2many fields",
-    "version": "14.0.1.0.1",
+    "version": "15.0.1.0.1",
```
Comment (gaikaz) [inline web_widget_x2many_2d_matrix/__manifest__.py]:
```suggestion
    "version": "15.0.1.0.0",
```

**Review by gaikaz — COMMENTED** [review]
Code review looks good.
I would also like to do a functional review on runboat, but the build is now gone. Should rebuild when you change the version here.
Also, you should probably squash the later commits to the first one named
`[15.0][MIG] web_widget_x2many_2d_matrix`

**Comment by ChrisOForgeFlow** [comment]
> Code review looks good. I would also like to do a functional review on runboat, but the build is now gone. Should rebuild when you change the version here. Also, you should probably squash the later commits to the first one named `[15.0][MIG] web_widget_x2many_2d_matrix`

Done!, thanks for review

**Review by LoisRForgeFlow — APPROVED** [review]
Tested, works as expected

**Comment by LoisRForgeFlow** [comment]
@gaikaz commits squashed

**Review by etobella — APPROVED** [review]

**Comment by gaikaz** [comment]
I couldn't get runboat to load a couple of times after the squash. Sorry, for not saying anything.
But I see it just got merged :) Good work then.

**Pull request merged**